README.rst

```diff
@@ -121,6 +121,24 @@ Prepare view sets to import/export via API
         resource_class = resources.BookResource
 
 
+If you use multiple or custom admin site, register required models to them
+
+.. code-block:: python
+
+    # app/admin.py
+    from import_export_extensions.models import ExportJob
+    from import_export_extensions.admin.model_admins.export_job_admin import ExportJobAdmin
+
+    from django.apps import AppConfig
+
+    custom_admin_site = BaseAdminSite(name="custom_admin")
```

Comment:
Seems like `BaseAdminSite` is not defined here

Please, add definition or import of `BaseAdminSite` to make the example more realistic (should be obvious it `BaseAdminSite` is existing class or defined by user)